configs/akeneo-api.json

```diff
@@ -1,15 +1,19 @@
 {
   "index_name": "akeneo-api",
   "start_urls": [
-    "https://api.akeneo.com/documentation.html"
+    "https://api.akeneo.com/getting-started.html",
+    "https://api.akeneo.com/getting-started-admin.html",
+    "https://api.akeneo.com/getting-started-admin.html",
```

Comment:
There is also a `stop_urls` that could be useful: https://github.com/algolia/docsearch-configs#stop_urls